src/cursor/cur_backup_incr.c

```diff
@@ -144,18 +145,7 @@ __curbackup_incr_next(WT_CURSOR *cursor)
         F_SET(cb, WT_CURBACKUP_INCR_INIT);
         __wt_cursor_set_key(cursor, 0, size, WT_BACKUP_FILE);
     } else {
-        if (F_ISSET(cb, WT_CURBACKUP_INCR_INIT)) {
-            /* Look for the next chunk that had modifications.  */
-            while (cb->bit_offset < cb->nbits)
-                if (__bit_test(cb->bitstring.mem, cb->bit_offset))
-                    break;
-                else
-                    ++cb->bit_offset;
-
-            /* We either have this object's incremental information or we're done. */
-            if (cb->bit_offset >= cb->nbits)
-                WT_ERR(WT_NOTFOUND);
-        } else {
+        if (!F_ISSET(cb, WT_CURBACKUP_INCR_INIT)) {
```

Comment:
Can you provide a bit more context to help me understand this change?  We used to walk the bitmap `if (F_ISSET(cb, WT_CURBACKUP_INCR_INIT))`. But now we walk the bitmap regardless of whether it's set?

Reply:
The original code would read in and initialize the information. It would then drop to setting the key to (0 (offset), granularity (len), WT_BACKUP_RANGE). The next time it would come in it would take the while-loop path. With consolidation it was also doing that and not consolidating until the next cursor->next call.

Now, if it is not initialized, we initialize it. So we walk the bitmap after successfully initializing it. If it is an error or one of the special cases we deal with it. Otherwise we now go to the already-initialized loop and consolidate what we can right from the start. Let me know if this is not clear and you want to talk more.